code/modules/mob/mob_helpers.dm

```diff
@@ -586,4 +600,19 @@ proc/is_blind(A)
 
 	return ..()
 
-
+/mob/proc/get_sex()
+	return gender
+
+//Tries to find the mob's email.
+/proc/find_email(real_name)
+	for(var/mob/mob in living_mob_list)
+		if(mob.real_name == real_name)
+			if(!mob.mind)
+				return
+			return mob.mind.initial_email_login["login"]
+
+/proc/get_both_hands(mob/living/carbon/M)
+	if(!istype(M))
+		return
+	var/list/hands = list(M.l_hand, M.r_hand)
+	return hands
```

Comment:
1) That proc should be in `inventory.dm`
2) It shouldn't direct access vars.
3) It should be called like `get_all_hands`

Reply:
Entire mob folder will be replaced in baymed no point to fix it now